src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,132 @@
+package wolox.training.models;
+
+import static wolox.training.utils.ErrorConstants.BOOK_ALREADY_OWNED;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name = "t_user")
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "USER_SEQ")
+    @SequenceGenerator(name = "USER_SEQ", sequenceName = "USER_SEQ")
+    private Long id;
+
+    @NotNull
+    @Column(nullable = false)
+    private String username;
+
+    @NotNull
+    @Column(nullable = false)
+    private String name;
+
+    @NotNull
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @NotNull
+    @Column(nullable = false)
+    @ManyToMany(cascade = {CascadeType.PERSIST, CascadeType.REFRESH})
```

Comment:
I would change persist for merge.

You can look at MERGE as the instruction to "merge" changes on the model back to the database.
But we're lacking the one that brings modifications iof the database to the model. And that's REFRESH :)

Reply:
My mistake, you said before that should be MERGE but I remembered like PERSIST. I'll fix it